test/System.Net.Http.Formatting.NetCore.Test/System.Net.Http.Formatting.NetCore.Test.csproj

```diff
@@ -9,15 +9,22 @@
     <DefineConstants>$(DefineConstants);NETFX_CORE</DefineConstants>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <!-- Version 2.4.5 does not support netstandard2.1. -->
+    <_VSRunnerVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">2.4.3</_VSRunnerVersion>
```

Comment:
Why not just use 2.4.3 everywhere?

Reply:
My aim is to use the latest versions everywhere I could. I also plan to expand the testing matrix to include `net6.0` and / or `net7.0`. Bottom line, `netcoreapp2.1` is a special case and I'd rather make that clear.